modules/mobile/app/serializers/mobile/v0/claim_serializer.rb

```diff
@@ -8,7 +8,12 @@ class ClaimSerializer
       attributes :date_filed, :min_est_date, :max_est_date, :phase_change_date, :open,
                  :waiver_submitted, :documents_needed, :development_letter_sent, :decision_letter_sent, :phase,
                  :ever_phase_back, :current_phase_back, :requested_decision, :claim_type, :contention_list,
-                 :va_representative, :events_timeline
+                 :va_representative, :events_timeline, :claim_type_code
+
+      attribute :claim_type_code do |data|
+        data.attributes[:claim_type_code] || nil
```

Comment:
rather then adding a field that will be nil to the EVSS model and serializer, I'm forcing it to be nil here if it doesn't exist.